security-privacy-questionnaire.md

```diff
@@ -6,36 +6,35 @@ Answering questions from [the questionaire doc here](https://w3ctag.github.io/se
+This specification also exposes camera movement related capabilities and current settings of a camera (pan, tilt and zoom) to a Web site provided that the user has granted pan-tilt-zoom camera permission to the said camera to the Web site. This allows the Web site to provide corresponding UI input controllers and to create web based conferencing systems.
 
 ### 02. Is this specification exposing the minimum amount of information necessary to power the feature?
-This specification does expose a list of capabilities usually found in a webcam as constrainable properties.
+Yes. This specification does expose a list of capabilities usually found in a webcam as constrainable properties.
 
 ### 03. How does this specification deal with personal information or personally-identifiable information or information derived thereof?
-This specification does not process personal information or personally-identifiable information.
+This specification does not deal with personal information or personally-identifiable information.
 
 ### 04. How does this specification deal with sensitive information?
 This specification does not share additional sensitive information such as camera serial numbers or such to the Web.
 
 That being said, this specification extends the [Media Capture and Streams](https://w3c.github.io/mediacapture-main/) specification which itself exposes video and audio streams from the user or from the user environment to Web sites with proper permissions.
 
-This specification also allows Web sites to pan, tilt and zoom cameras which may allow the video stream to expose objects and subjects which were originally out of the original field of view or which were originally too tiny to be distinguishable but could be enlarged by optical zoom. Therefore, these constrainable properties are behind an additional pan-tilt-zoom camera permission and cannot be accessed with normal camera permission alone.
+This specification also allows Web sites to pan, tilt and zoom cameras which may allow video streams to expose objects and subjects which were originally out of original field of views or which were originally too tiny to be distinguishable but could be enlarged by optical zooms. Therefore, these constrainable properties are behind an additional pan-tilt-zoom camera permission and cannot be accessed with normal camera permission alone.
 
 ### 05. Does this specification introduce new state for an origin that persists across browsing sessions?
 No.
 
-
 ### 06. What information from the underlying platform, e.g. configuration data, is exposed by this specification to an origin?
-No particular additional information is exposed about the camera hardware, like brand, etc, however we do expose if the platform has support for the constainable properties. 
+No particular additional information is exposed about the camera hardware, like brand, etc, however we do expose if the platform has support for the constainable properties.
```

Comment:
I would add:

_"A site with permission to one or more cameras will be able to probe the existence of these new capabilities on those cameras, by passing constraints to `getUserMedia` or `track.applyConstraints`. The new capabilities are:"_ (and then list them).

Also:

_"A site without camera permission may be able to probe the lack of a capability from the user's total set of cameras, but by doing so risks causing a user-facing prompt if the capability is available. This helps provide browsers with information about a site's camera needs when prompting the user, while deterring tracking libraries from attempting to use the API without detection."_

About pan-tilt-zoom permission: if I'm reading the spec right, the mere mention of `pan`, `tilt` or `zoom` in a constraints-set should trigger a prompt for pan-tilt-zoom permission, even if it ends up finding zero devices, is that correct? Is that how it's implemented in Chrome or other browsers? If so, then there's no added pre-prompt exposure from those three properties, otherwise there would be.

Reply:
Chrome does not request a pan-tilt-zoom camera permission (but only a normal camera permission) if there are no pan-tilt-zoom cameras.

Reply:
Isn't _track.applyConstraints_ an overly complex method to probe existence of capabilities when there are _track.getCapabilities_ and _track.getSettings_ for exacly that kind of probing?